geoai/universat.py

```
@@ -0,0 +1,230 @@
+        UniverSat.from_pretrained(model_name_or_path, **kwargs)
+        if pretrained
+        else hubconf.universat(pretrained=False, size=size, **kwargs)
+    )
+    model = model.to(device or get_device())
+    return model.eval() if eval_mode else model
+
+
+class UniverSatProcessor:
+    """Processor class for UniverSat, handling GeoTIFFs, normalization, and inference."""
+
+    def __init__(
```

Ruff (0.15.21)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)